website/addons/wiki/templates/edit.mako

```diff
@@ -74,12 +74,12 @@
                class="${'col-sm-{0}'.format(12 / num_columns)}"
                style="${'' if 'view' in panels_used else 'display: none'}">
               <div class="osf-panel panel panel-default no-border" data-bind="css: { 'no-border reset-height': $root.singleVis() === 'view', 'osf-panel-flex': $root.singleVis() !== 'view' }">
-                <div class="panel-heading wiki-panel-header wiki-single-heading" data-bind="css: { 'osf-panel-heading-flex': $root.singleVis() !== 'view', 'wiki-single-heading': $root.singleVis() === 'view' }">
-                    <div class="row">
-                        <div class="col-sm-4">
-                            <span class="panel-title" > <i class="fa fa-eye"> </i>  View</span>
+                <div class="panel-heading wiki-panel-header wiki-single-heading" data-bind="css: { 'osf-panel-heading-flex': $root.singleVis() !== 'view', 'wiki-single-heading': $root.singleVis() === 'view' }" style=" padding-top: 0; padding-bottom: 0;">
+                    <div class="row" style="position: relative; top: 50%; transform: translateY(-50%);">
```

Comment:
Can you put this into a CSS class `website/static/css/pages/profile-page.css`? Also, does translateY work in IE 9? That seems very new-fangled.